tests/App/Models/Users.php

```diff
@@ -29,7 +29,7 @@ public function create($fields = [], Validation $extra = null)
     {
         $model = $this;
 
-        $this->di->hook('model.before.create', function ($model) {
+        $this->di->hook('model.before.create.' . get_called_class(), function ($model) {
```

Comment:
Could you remove clearHooks too? https://github.com/ice/framework/pull/209/files#diff-5507e26bff1e1fbfec725771c7a56f84R39